andela/hc-silicon
# #157694765 Add Third Party Checks Integration

#### What does this PR do?
Allows users to create alerts for a series of third party online business automation applications
#### Description of Task to be completed?
GIVEN as a user I am running scheduled tasks on third party applications
WHEN I find the application is supported by HealthChecks
THEN I can create alerts for that application
#### How should this be manually tested?
- Clone the repo into your local machine.
- cd into the project.
- set up the virtualenv and run pip install -r requirements.txt.
- Run the server using python manage.py runserver
- Log in and head to ```third-party-checks``` tab and click ```Authorize GitHub integration```
- Enter GitHub credentials and get redirected to HealthChecks where you can choose a repository to create a check to.
- Head over to `checks` tab to view your created check.
#### Any background context you want to provide?
- You can navigate to Settings > Webhooks on your GitHub repo to view the created webhook.
#### What are the relevant pivotal tracker stories?
[157694765
](https://www.pivotaltracker.com/story/show/157694765)
#### Screenshots
![image](https://user-images.githubusercontent.com/31405837/41741607-4329fdaa-75a4-11e8-8069-99d316073807.png)
![image](https://user-images.githubusercontent.com/31405837/41741624-49ed822e-75a4-11e8-8ac5-f55dc896724f.png)
![image](https://user-images.githubusercontent.com/31405837/41741630-51389ae6-75a4-11e8-99bf-e1940370f60c.png)
![image](https://user-images.githubusercontent.com/31405837/41741640-56607c78-75a4-11e8-9367-8756d9b636d8.png)
![image](https://user-images.githubusercontent.com/31405837/41741650-5cec33fc-75a4-11e8-9515-be44304df90b.png)
![image](https://user-images.githubusercontent.com/31405837/41741657-622807ce-75a4-11e8-821c-a9a2ca8b08b6.png)
![image](https://user-images.githubusercontent.com/31405837/41741670-67cbf2a8-75a4-11e8-9a1c-cd0cda6d37ac.png)
![image](https://user-images.githubusercontent.com/31405837/41741676-6d0a1826-75a4-11e8-817b-907885c45249.png)
